core/ErrorHandler.php

```diff
@@ -123,7 +123,7 @@ private static function getHtmlMessage($errno, $errstr, $errfile, $errline, $tra
         $message = ErrorHandler::getErrNoString($errno) . ' - ' . $errstr;
 
         $html = "<p>There is an error. Please report the message (Matomo " . (class_exists('Piwik\Version') ? Version::VERSION : '') . ")
-        and full backtrace in the <a href='?module=Proxy&action=redirect&url=https://forum.matomo.org' target='_blank'>Matomo forums</a> (please do a search first as it might have been reported already!).</p>";
+        and full backtrace in the <a target='_blank' rel='noreferrer' href='https://forum.matomo.org'>Matomo forums</a> (please do a search first as it might have been reported already!).</p>";
```

Comment:
I reckon for security be also good to always add `rel=noopener` but not needed.

Reply:
Good idea. I'll look into that.